src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/RepairPatch.java

```diff
@@ -188,10 +192,14 @@ private static int overfittingSort(RepairPatch patch1, RepairPatch patch2, Featu
 	}
 
 	private ODSLabel computeODSLabel(int patchId, Long buildId) {
+		
+		ODSLabel label = ODSLabel.UNKNOWN;
+
+		try {		
```

Comment:
The file seems not indented: The try and the block has the same indentation.
Even I am not sure if in Repairnator we use a particular code style, we can use by default that one from Eclipse, which has one by default.

Reply:
Ah, thanks Matias for the kindly remind. I made another commit to improve the indention.